de.metas.handlingunits.base/src/main/sql/postgresql/system/5501400_cli_gh4567_Locked_Attribute_PackingInstruction.sql

```diff
@@ -0,0 +1,10 @@
+-- 2018-09-13T17:08:21.083
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO M_HU_PI_Attribute (AD_Client_ID,AD_Org_ID,Created,CreatedBy,HU_TansferStrategy_JavaClass_ID,IsActive,IsDisplayed,IsInstanceAttribute,IsMandatory,IsOnlyIfInProductAttributeSet,IsReadOnly,M_Attribute_ID,M_HU_PI_Attribute_ID,M_HU_PI_Version_ID,PropagationType,SeqNo,Updated,UpdatedBy,UseInASI) VALUES (1000000,1000000,TO_TIMESTAMP('2018-09-13 17:08:20','YYYY-MM-DD HH24:MI:SS'),100,540027,'Y','Y','N','N','N','Y',540039,540040,100,'NONE',9040,TO_TIMESTAMP('2018-09-13 17:08:20','YYYY-MM-DD HH24:MI:SS'),100,'N')
```

Comment:
I added them with client and org 1000000 because all the other M_HU_PI_Attribute entries are with org and client 100000 ( except "Gewicht Brutto kg" and "Gewicht Tara", which have org 0) so I thoguht it's our standard.